react-native/README.md

```diff
@@ -42,6 +45,12 @@ npm run watch
 
 #### iOS
 
+Install cocoapods:
+
+```sh
+sudo gem install cocoapods --pre -v 1.0.0.beta.6
```

Comment:
Yeah I personally used `--user-install` but didn't want to have to include the details. Could drop the sudo to match the `npm install -g` and let the user figure it out :smirk: